lms/templates/dashboard/_dashboard_status_verification.html

```diff
@@ -6,58 +6,39 @@
 %>
 
 %if verification_status == 'approved':
-<li class="status status-verification is-accepted">
-  <span class="title status-title">${_("ID Verification Status")}</span>
+    <li class="status status-verification is-accepted">
+        <span class="title status-title">${_("Verification Status: Approved")}</span>
 
-  <div class="status-data">
-    <span class="status-data-message">${_("Reviewed and Verified")}</span>
-
-    <div class="status-note">
-      <span class="deco-arrow"></span>
-
-      <p>${_("Your verification status is good for one year after submission.")}</p>
-    </div>
-  </div>
-</li>
+        <div class="status-note">
+            <p>${_("Your edX Verification is reviewed and approved. Your verification status is good for one year after submission.")}</p>
+        </div>
+    </li>
 %endif
 
 %if verification_status == 'pending':
-<li class="status status-verification is-pending">
-  <span class="title status-title">${_("ID Verification Status")}</span>
-
-  <div class="status-data">
-    <span class="status-data-message">${_("Pending")}</span>
+    <li class="status status-verification is-pending">
+        <span class="title status-title">${_("Verification Status: Pending")}</span>
 
-    <div class="status-note">
-      <span class="deco-arrow"></span>
-
-      <p>${_("Your verification photos have been submitted and will be reviewed shortly.")}</p>
-    </div>
-  </div>
-</li>
+        <div class="status-note">
+            <p>${_("Your edX Verification is pending. Your verification photos have been submitted and will be reviewed shortly.")}</p>
+        </div>
+    </li>
 
 %endif
 
 %if verification_status in ['must_reverify', 'expired']:
-<li class="status status-verification is-denied">
-<span class="title status-title">${_("{platform_name} Verification Status").format(platform_name=static.get_platform_name())}</span>
+    <li class="status status-verification is-denied">
+        <span class="title status-title">${_("Verification Status: Expired")}</span>
 
-  <div class="status-data">
-    <span class="status-data-message">${verification_msg}</span>
+        <div class="status-note">
+            <p>${_("Your edX Verification has expired. To receive a verified certificate, you have to submit a new photo of yourself and your government-issued photo ID before the course ends.")}</p>
+        </div>
```

Comment:
Does this need to be a ```p``` inside of a ```div```? Can it not just be a ```p``` with class status-note```?